docker/entrypoint.sh

```diff
@@ -1,14 +1,11 @@
 #!/bin/ash -e
 
-#mkdir -p /var/log/supervisord/ /var/log/php8/ \
-
-## check for .env file and generate app keys if missing
-if [ -f /pelican-data/.env ]; then
```

Comment:
This should stay so if they mess up the mount it fails

Reply:
The .env is symlinked to the mounted .env, so assuming they haven't done anything special, it will still fail. I did that to allow the .env to be overwritten by its own mount if anyone desires, while still retaining the original functionality.